modules/system/common/hardware/powersave/power-tuning.sh

```
@@ -0,0 +1,325 @@
+}
+
+set_pcie_aspm() {
+    local policy=$1
+    local available_policies
+
+    if [[ ! -f /sys/module/pcie_aspm/parameters/policy ]]; then
+        log "PCIe ASPM not available"
+        return 1
+    fi
+
+    available_policies=$(cat /sys/module/pcie_aspm/parameters/policy | tr -d '[]')
```

Shellcheck (0.10.0)

[style] Useless cat. Consider 'cmd < file | ..' or 'cmd file | ..' instead.

(SC2002)